tests/gen-name-dir-key-test.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var assert = require('assert');
+var genNameDirKey = require('../lib/gen-name-dir-key');
+
+describe.skip('gen-name-dir-key-test', function() {
```

Comment:
@dcombslinkedin can you implement these tests

Reply:
Done, in resolve-package-path-test.js. Checking in shortly.